packages/gestalt/src/SearchField.js

```diff
@@ -107,7 +107,6 @@ export default class SearchField extends React.Component<Props, State> {
         onMouseLeave={this.handleMouseLeave}
         onFocus={this.handleFocus}
         onBlur={this.handleBlur}
-        color="white"
```

Comment:
This should be fine but can you show before and after screenshots of this change in the PR description? Also, do you mind adding a line entry to the CHANGELOG.md file @Neet-Nestor

Reply:
Done. Thanks!